release/_publishing.sh

```diff
@@ -166,17 +166,44 @@ function upload_to_docker_hub {
 }
 
 function _update_bundles_yml {
-	if (yq eval ".quarterly | has(\"${_PRODUCT_VERSION}\")" "${BASE_DIR}/bundles.yml" | grep -q "true")
-	then
-		lc_log INFO "The ${_PRODUCT_VERSION} product version was already published."
+	local product_version_key="$(echo "${_PRODUCT_VERSION}" | cut -d'-' -f 1)"
 
-		return "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}"
-	fi
+		if (yq eval ".quarterly | has(\"${_PRODUCT_VERSION}\")" "${BASE_DIR}/bundles.yml" | grep -q "true") ||
+		(yq eval ".\"${product_version_key}\" | has(\"${_PRODUCT_VERSION}\")" "${BASE_DIR}/bundles.yml" | grep -q "true")
+		then
+			lc_log INFO "The ${_PRODUCT_VERSION} product version was already published."
 
-	local latest_key=$(yq eval ".quarterly | keys | .[-1]" "${BASE_DIR}/bundles.yml")
+			return "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}"
+		fi
 
-	yq --indent 4 --inplace eval "del(.quarterly.\"${latest_key}\".latest)" "${BASE_DIR}/bundles.yml"
-	yq --indent 4 --inplace eval ".quarterly.\"${_PRODUCT_VERSION}\".latest = true" "${BASE_DIR}/bundles.yml"
+	case "${_PRODUCT_VERSION}" in
+		*q*)
+			local latest_key=$(yq eval ".quarterly | keys | .[-1]" "${BASE_DIR}/bundles.yml")
+
+			yq --indent 4 --inplace eval "del(.quarterly.\"${latest_key}\".latest)" "${BASE_DIR}/bundles.yml"
+			yq --indent 4 --inplace eval ".quarterly.\"${_PRODUCT_VERSION}\".latest = true" "${BASE_DIR}/bundles.yml"
+			;;
+		7.3.*)
+			yq --indent 4 --inplace eval ".\"${product_version_key}\".\"${_PRODUCT_VERSION}\" = {}" "${BASE_DIR}/bundles.yml"
+			;;
+		*u*)
+			local nightly_bundle_url=$(yq eval ".\"${product_version_key}\".\"${product_version_key}.nightly\".bundle_url" "${BASE_DIR}/bundles.yml")
+
+			yq --indent 4 --inplace eval "del(.\"${product_version_key}\".\"${product_version_key}.nightly\")" "${BASE_DIR}/bundles.yml"
+			yq --indent 4 --inplace eval ".\"${product_version_key}\".\"${_PRODUCT_VERSION}\" = {}" "${BASE_DIR}/bundles.yml"
+			yq --indent 4 --inplace eval ".\"${product_version_key}\".\"${product_version_key}.nightly\".bundle_url = \"${nightly_bundle_url}\"" "${BASE_DIR}/bundles.yml"
+			;;
+		*ga*)
+			local ga_bundle_url="releases-cdn.liferay.com/portal/${_PRODUCT_VERSION}/"$(curl -fsSL "https://releases-cdn.liferay.com/portal/${_PRODUCT_VERSION}/.lfrrelease-tomcat-bundle")
+
+			sed -i '0,/latest: true/s// /' "${BASE_DIR}/bundles.yml"
+
+			sed -i "/7.4.13:/i ${product_version_key}:" "${BASE_DIR}/bundles.yml"
```

Comment:
@kiwm, can you revisit this code and try to optimize using a regex like `latest: true(?!7.4.13:)`?